test/factories/code_schools.rb

```diff
@@ -8,6 +8,6 @@
     has_online Faker::Boolean.boolean
     online_only Faker::Boolean.boolean
     notes Faker::Lorem.paragraph
-    mooc Faker::Boolean.boolean
+    mooc true
```

Comment:
I just merged a PR that fixes this issue:  https://github.com/OperationCode/operationcode_backend/pull/261

Please rebase on master, and then change this to `mooc false`, and retest.  You should be good to go at that point.

Reply:
and done! tests check out, I changed mooc back to a faker boolean.